backbone/src/main/res/layout/rsb_image_capture_step_layout.xml

```diff
@@ -45,6 +45,8 @@
         style="@style/rsb_image_capture_step_style"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/imageCaptureStepQuestionItself"
+        android:focusable="true"
+        android:contentDescription="@null"
```

Comment:
shouldn't this contentDescription point to a `@string` explaining what this field is about? I know in "code" you then touch some of these, but at the very least, (and honestly at the most) I was not necessarily expecting a dynamic solution, rather a series of generic strings saying generic things about what the field is. I think these should, by default, have a content description other than `null`. 

Since you are adding strings (with the pain that involves due to Multi-language), you may as well take the opportunity to do this as well. 

Am I missing something?

Reply:
I did it dynamically because the user needs to know which cell item does this image belong to, so I'm adding the step title for each image in the content description.
of course I can add this to the xml as well, I'll do that shortly